tests/test_string_exports.js

```diff
@@ -0,0 +1,25 @@
+// Test ES2020 string export/import names
+import { assert } from "./assert.js";
+import * as mod from "./fixture_string_exports.js";
+
+// Test string import names
+import { "string-export-1" as str1 } from "./fixture_string_exports.js";
+import { "string-export-2" as str2 } from "./fixture_string_exports.js";
+import { "string-name" as strMixed } from "./fixture_string_exports.js";
+
+// Test regular imports still work
+import { regularExport, normalName } from "./fixture_string_exports.js";
+
+// Verify values
+assert(str1 === "value-1");
```

Comment:
It's not wrong but it's better to use the two-arg (actual, expected) overload:

```suggestion
assert(str1, "value-1");
```